### ISSUE TITLE
NextGen still crashing if you select a graph from a simulation in progress

### ISSUE THREAD

**Issue by AllanPeake** [APSIMInitiative/ApsimX#3139]
If I am running a Chickpea simulation and inadvertently select a graph from within the simulation that is being run, APSIM crashes

**Comment by hol353** [comment]
No, probably can't wait that long. Put in a test somewhere in DataStore's GetData methods that checks to see if data is being written. If so return null.
